spec/property.dd

```diff
@@ -65,7 +65,7 @@ $(THEAD Property, Description)
 $(TROW $(RELATIVE_LINK2 classinfo, $(D .classinfo)), Information about the dynamic type of the class)
 )
 
-$(SECTION3 $(LNAME2 init, .init) Property,
+$(SECTION3 $(LNAME2 init, .init Property),
```

Comment:
This unifies this page (same headings contain "Property" in the link, some didn't).